pages/linux/protonvpn-cli-connect.md

```diff
@@ -0,0 +1,28 @@
+# protonvpn-cli connect
+
+> Official client to connect to ProtonVPN from the command-line.
+> More information: <https://protonvpn.com/support/linux-vpn-setup/>.
+
+- Connect to ProtonVPN interactively:
+
+`protonvpn-cli connect`
+
+- Connect to ProtonVPN using the fastest server available:
+
+`protonvpn-cli connect --fastest`
+
+- Connect to ProtonVPN using a specific server with a specific protocol:
+
+`protonvpn-cli connect {{server_name}} --protocol {{udp|tcp}}`
+
+- Connect to ProtonVPN using a random server and a specific protocol:
```

Comment:
```suggestion
- Connect to ProtonVPN using a random server with a specific protocol:
```

Reply:
Both make sense to me. I wouldn't pick 1 over the other